### PR TITLE
Add `cstool` in CMake Package Config + CMake Github Workflow

### PR DESCRIPTION
**Pull request by theblackunknown**
## Description

This PR add `cstool` to the list of `capstone-targets` so it can be used after a `find_package` with another CMake project.
As an added benefit, capstone integration in package manager like `vcpkg` can be improved.

Moreover, I have added simple CMake presets and CMake CI to test those changes in an automated way.

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: CMake
```
Comment (kabeor) [inline .github/workflows/cmake.yml:1]: Plz merge this file into CITest.yml

File: .github/workflows/cmake.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            platform: macos
+
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - uses: lukka/get-cmake@latest
+      name: Get CMake
+
+    - uses: ilammy/msvc-dev-cmd@v1
```
Comment (kabeor) [inline .github/workflows/cmake.yml:31]: For now the windows part in CITest.yml is commented out, can you fix that?

**Comment by theblackunknown** [comment]
Hey @kabeor thanks for your feedbacks !
I did not find the time to address those this week but I'll get back to it next week.

**Comment by theblackunknown** [comment]
@kabeor , I have upgraded `CITest.yml` for Windows like you suggested.
I am happy to help you enhance it further (e.g. run unit test on Windows) in another PR if that is helpful to you

**Comment by kabeor** [comment]
@theblackunknown nice work！I really very much welcome your contributions to capstone：）

**Comment by theblackunknown** [comment]
@kabeor Thanks !
Do you think this PR can be merged ? I could push another PR for CI Windows testing then

**Comment by kabeor** [comment]
merged, let's see if msvc test works well

**Pull request merged**